docs/home/contribute/participating.md

```diff
@@ -4,51 +4,73 @@ title: Participating in SIG-DOCS
+Any member of the [Kubernetes organization](https://github.com/kubernetes) can review a pull request, and SIG Docs team members frequently request reviews from other SIG members for technical accuracy. 
+You can indicate your approval by adding a comment of `/lgtm` to a pull request.
 
 - Reviewers
 
-Reviewers are individuals who have volunteered to provide their time to review documentation pull requests. You become a reviewer by adding your github handle to the [OWNERS file](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md) within the [website repo](https://github.com/kubernetes/website).
+Reviewers are individuals who review documentation pull requests. You can become a reviewer by adding your github handle to the [OWNERS file](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md) within the [website repo](https://github.com/kubernetes/website).
+(Additions to the OWNERS file require contributors to meet certain requirements. See the [Kubernetes community repo](https://github.com/kubernetes/community/blob/master/community-membership.md) for more information.
 
-Our automation will reviewers to use for reviews on pull requests, and a contributor may request your review with an `/assign [your_github_handle]` comment in the pull request. To indicate your approval of a pull request, a reviewer should add an `/approve` comment to the pull request. A reviewer may also use add a `/hold` comment on the pull request to prevent the pull request from being merged. Another reviewer or approver may also remove that hold with the comment `/hold cancel`.
+Our automation assigns reviewers to pull requests, and contributors can request your review with a comment on the pull request: `/assign [your_github_handle]`. To indicate that a pull request requires no further changes, a reviewer adds a comment to the pull request `/lgtm`.
 
-When a reviewer is assigned a pull request to review it is not a sole responsibility, and any other reviewer may also offer their opinions on the pull request. If a reviewer is requested, it is generally expected that the PR will be left to that reviewer to do their editorial pass on the content. For small and/or trivial pull requests, the assigned reviewer may be skipped if an approver thinks the content is sufficient and well understood. 
+Approvers can indicate their approval with a comment: `/approve`.
+
+Reviewers and approvers can add a `/hold` comment to prevent the pull request from being merged. Another reviewer or approver can remove a hold with the comment: `/hold cancel`.
+
+When both of a `lgtm` and `approval` tag have been added, and there are no `hold` tags, the automation on the repository will merge the pull request.
```

Comment:
"When both the `lgtm` and `approve` commands are added, and there are no `hold` tags, the automation on the repository merges the pull request."
(some edits for grammar, last one for future tense -> present ;))
Also maybe be consistent about how we refer to these critters? I see "comment" and "tag" here, and it's also a command issued to the bot ...